todo-app/src/components/DeleteButton.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+
+export default class DeleteButtom extends Component {
+    onDeletingAll = () => {
+        this.props.onDeletingAll()
+    }
+    render() {
+        return (
+            <div className="delete-container">
+                <button className="delete-all-button" onClick={ this.onDeletingAll }>
```

Comment:
IF there's no specific logic of your event in this component, there will be no need for creating new function. You can just use `this.props.onDeleteAll()` directly like the following.

```
<button className="delete-all-button" onClick={ this.props.onDeletingAll }>
```